specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/guestDiagnosticSettingsAssociation_API.json

```diff
@@ -0,0 +1,444 @@
+                "tags": [
+                    "GuestDiagnosticsSettingsAssociation"
+                 ],
+                 "x-ms-examples": {
+                    "Create or update an guest diagnostic settings association": {
+                        "$ref": "./examples/deleteGuestDiagnosticSettingsAssociation.json"
+                    }
+                },
+                "operationId": "GuestDiagnosticsSettingsAssociation_Delete",                
+                "parameters": [
+                    {
+                        "name": "resourceUri",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The fully qualified ID of the resource, including the resource name and resource type.",
+                        "x-ms-skip-url-encoding": true
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingsAssociationNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The diagnostic settings association was successfully deleted."
+                    },
+                    "204": {
+                        "description": "The diagnostic settings association does not exist. It may have already been deleted."
+                    },
+                    "default": {
+                        "description": "An error occurred and the diagnostic settings association could not be deleted.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "description": "Updates an existing guestDiagnosticsSettingsAssociation Resource. To update other fields use the CreateOrUpdate method",
+                "operationId": "guestDiagnosticsSettingsAssociation_Update",
+                "parameters": [
+                    {
+                        "name": "resourceUri",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The fully qualified ID of the resource, including the resource name and resource type.",
+                        "x-ms-skip-url-encoding": true
+                    },
+                    {
+                      "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingsAssociationNameParameter"
+                    },
+                    {
+                        "name": "diagnosticSettingsAssociation",
```

Comment:
Is there are reason you revert the name from "parameters" here? It's not an issue, but I'm curious.

Reply:
I do not remember. I have changed it back,

Reply:
Now I know - it caused Model validation error. Done correctly now :)